### DISCUSSION TITLE
Discussion: [A scammer is copying this service and website]

### DISCUSSION THREAD

**Discussion by Remzi1993** [janhq/jan#2614] [General]
**Motivation**
To warn everyone
**Discussion**
See scam and phishing website: https://janproai.com
And they are advising on Reddit, see: https://www.reddit.com/user/Ivy-Robinson/comments/1bu7wpp/meet_8k_image_generation/

![Screenshot 2024-04-03 211218](https://github.com/janhq/jan/assets/980950/0fc3c8df-f8e5-4113-9fe5-42ff1c3d5518)

**Comment by Van-QA** [comment]
I will convert this to discussion, thank everyone 🙏
